### PR TITLE
update security policy supported versions

### PR DESCRIPTION
**Pull request by jchappelow**

### CODE REVIEW

**Review by brennanjl — DISMISSED** [review]

**Comment by brennanjl** [comment]
So I guess no lts for v0.8?

**Comment by brennanjl** [comment]
Nbd IMO, but curious why

**Comment by jchappelow** [comment]
I updated it with 0.8 yesterday after chat.  Will you verify again that the table is what we want?

File: SECURITY.md [hunk 1]
```diff
@@ -5,8 +5,9 @@
 | Version | Supported |
 | ------- | --------- |
 | Latest beta or release candidate    | ✅        |
+| v0.9.x  | ✅        |
 | v0.8.x  | ✅        |
-| < v0.8  | ❌        |
+| < v0.9  | ❌        |
```
Comment (brennanjl) [inline SECURITY.md:10]: Seems incorrect, this says that we do not support anything less than v0.9. It should be `< v0.8`

Comment (jchappelow) [reply SECURITY.md:10]: derp oh yeah sorry

**Review by brennanjl — APPROVED** [review]

**Pull request merged**